packages/cli/internal/pkg/osutils/utils.go

```diff
@@ -0,0 +1,51 @@
+package osutils
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/aws/amazon-genomics-cli/internal/pkg/cli/clierror/actionableerror"
+)
+
+var osUserHomeDir = os.UserHomeDir
+
+// DetermineHomeDir returns the file system directory where the AGC files live.
+func DetermineHomeDir() (string, error) {
+	dir, err := osUserHomeDir()
+	if err != nil {
+		return "", actionableerror.New(err, "Please check that your home or user profile directory is defined within your environment variables")
+	}
+	return dir, nil
+}
+
+// ExpandHomeDir returns the expanded home directory path for the current user
+func ExpandHomeDir(rootPath string) (string, error) {
+	if rootPath == "~" {
+		homeDir, _ := DetermineHomeDir()
```

Comment:
Why are we ignoring error in this case?

Reply:
Ooops must be slipped after rebasing the branch. It's changed now.